test/safe.test.js

```diff
@@ -94,15 +87,10 @@ describe('Safe', () => {
       // Let's make the trust connections needed to get the Safe deployed
       await Promise.all(
         predeployedSafes.map((predeployedAddress, index) =>
-          addTrust(
-            {
-              account: accounts[index + 1],
-              safeAddress: predeployedAddress,
-              safeAddressToTrust: safeAddress,
-              limitPercentage: 50,
-            },
-            core,
-          ),
+          core.trust.addConnection(accounts[index + 1], {
```

Comment:
The `addTrust` method was added because the `addConnection` functionality did not exist yet and I needed a way to be able to test that checking 3 trust connections requirement was working. Now the method exists so i can do the testing properly, that is why i removed it.